sympy/physics/mechanics/rigidbody.py

```diff
@@ -266,65 +255,13 @@ def kinetic_energy(self, frame):
 
         """
 
-        rotational_KE = (self.frame.ang_vel_in(frame) & (self.central_inertia &
-                self.frame.ang_vel_in(frame)) / sympify(2))
-
-        translational_KE = (self.mass * (self.masscenter.vel(frame) &
-            self.masscenter.vel(frame)) / sympify(2))
-
+        rotational_KE = dot(self.frame.ang_vel_in(frame), dot(
+            self.central_inertia, self.frame.ang_vel_in(frame)) / sympify(2))
+        translational_KE = (self.mass * dot(
+            self.masscenter.vel(frame), self.masscenter.vel(frame)) / sympify(2)
+                            )
```

Comment:
I think this is easier to read. Would require import of `S.Half`.
```suggestion
        translational_KE = S.Half * self.mass * dot(
            self.masscenter.vel(frame), self.masscenter.vel(frame)
        )
                            
```